src/App.jsx

```diff
@@ -14,33 +16,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'alphabet';
+const SORT_LENGTH = 'length';
+const SORT_REVERSE = 'reverse';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function sortGoods(goods, isLight, isReverse) {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (isLight) {
+    preparedGoods.sort((good1, good2) => {
+      switch (isLight) {
+        case SORT_ALPHABET:
+          return good1.localeCompare(good2);
+        case SORT_LENGTH:
+          return good1.length - good2.length;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReverse) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
+export const App = () => {
+  const [isLight, setIsLight] = useState('');
+  const [isReverse, setIsReverse] = useState('');
```

Comment:
```suggestion
  const [isReverse, setIsReverse] = useState(false);
```
you can just save the boolean value here